crates/swc_ecma_transforms_compat/src/es2015/spread.rs

```diff
@@ -251,9 +251,11 @@ impl Spread {
             };
         }
 
+        let mut arg_list = vec![];
         for arg in args {
             if let Some(arg) = arg {
                 let ExprOrSpread { expr, spread } = arg;
+                arg_list.push(expr.clone().as_arg());
```

Comment:
This clone seems redundant.

Reply:
`expr`used without `clone`method will cause `use of moved value` error in next `*expr` location.

--- 
Error  stack  is as follows.
```shell
error[E0382]: use of moved value: `*expr`
   --> crates/swc_ecma_transforms_compat/src/es2015/spread.rs:286:34
    |
258 |                 arg_list.push(expr.as_arg());
    |                                    -------- `*expr` moved due to this method call
...
286 |                         buf.push(match *expr {
    |                                  ^^^^^^^^^^^ value used here after move
    |
    = note: move occurs because `*expr` has type `swc_ecma_ast::Expr`, which does not implement the `Copy` trait
help: you can `clone` the value and consume it, but this might not be your desired behavior
    |
258 |                 arg_list.push(expr.clone().as_arg());
    |                                    ++++++++

For more information about this error, try `rustc --explain E0382`.
error: could not compile `swc_ecma_transforms_compat` (lib) due to previous error
```
---
```shell
$ rustc --version
rustc 1.70.0-nightly (da7c50c08 2023-03-19)
```